test/functional/apps/dashboard/_dashboard.js

```diff
@@ -31,6 +31,7 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.dashboard.clickNewDashboard();
       await dashboardVisualizations.createAndAddTSVBVisualization('TSVB');
       await PageObjects.dashboard.addVisualizations(PageObjects.dashboard.getTestVisualizationNames());
+      await PageObjects.common.clearAllToasts();
```

Comment:
I know you said in Slack that having the clearAllToasts call later didn't work, but I can't understand why it wouldn't.  I tried it locally at the very end of this test and it cleared both the toast messages instead of just the one you're clearing here.  In fact I was able reproduce the failure by changing the window size to a height of 600 when only this visualization notification is cleared and the "Saved search was added..." message was still blocking the timepicker.
I think it makes more sense to clear all the toast messages after the createAndAddSavedSearch call so that there's no notifications at the start of the next test.

Reply:
For me the bulk of the toast messages occur right after line 33 so that is what needed to be cleared, but if it works for you some other way just let me know which line number to put it in

Reply:
I moved it to line 44 I think that is what you wanted, but if not just let me know which line number

Reply:
For me about 10 or so notifications are all cleared after step 33 not just one notificaiton, but bulk of them, so I am not sure what you are seeing.  If you can have debug on you will see where in the code it clears all the messages and for me it is after step 33 in the code.  I may move it back to line 33, so you will have to show me your execution.